### PR TITLE
chore(CI): CI setup

### PR DESCRIPTION
**Pull request by BrantalikP**
- setup CODEOWNERS file - https://github.com/torolocos/react-rtc/issues/5
- setup Pull request template
- Install + setup Eslint and prettier
- setup TS and ESLINT CI checks
- setup editconfig file
- Publishing is triggered only on new release

### CODE REVIEW

File: packages/react-rtc/.editorconfig [hunk 1]
```diff
@@ -0,0 +1,12 @@
+# EditorConfig is awesome: https://EditorConfig.org
```
Comment (pchalupa) [inline packages/react-rtc/.editorconfig:1]: This config could be at the root of monorepo.

File: packages/react-rtc/.eslintrc.js [hunk 2]
```diff
@@ -0,0 +1,32 @@
+module.exports = {
```
Comment (pchalupa) [inline packages/react-rtc/.eslintrc.js:1]: Could you please use JSON? https://eslint.org/docs/user-guide/configuring/configuration-files#configuration-file-formats

File: packages/react-rtc/package.json [hunk 3]
```diff
@@ -1,25 +1,39 @@
...
+  "peerDependencies": {
+    "react": "^17.0.2"
+  },
+  "devDependencies": {
+    "@types/react": "^17.0.37",
+    "@typescript-eslint/eslint-plugin": "^5.15.0",
+    "@typescript-eslint/parser": "^5.15.0",
+    "eslint": "^8.11.0",
+    "eslint-config-airbnb": "^19.0.4",
+    "eslint-config-google": "^0.14.0",
```
Comment (pchalupa) [inline packages/react-rtc/package.json:30]: We should use just one of them.

File: .github/workflows/npm-publish.yml [hunk 4]
```diff
@@ -1,23 +1,23 @@
-name: "🚀 publish"
+name: '🚀 publish'
 
 on:
-  push:
+  release:
```
Comment (pchalupa) [inline .github/workflows/npm-publish.yml:4]: If `bump-version` workflow adds tag, then I think this could be on push.

**Review by pchalupa — CHANGES_REQUESTED** [review]
Nice work! Also we could define Turbo pipelines https://turborepo.org/docs/features/pipelines to get linter running with turbo.

File: .github/workflows/npm-publish.yml (hunk 4 above)
Comment (BrantalikP) [reply .github/workflows/npm-publish.yml:4]: keep push

**Review by pchalupa — APPROVED** [review]

**Review by pchalupa — APPROVED** [review]

**Pull request merged**